sources/packages/api/src/database/migrations/1652466821171-CleanRestrictionsTable.ts

```diff
@@ -0,0 +1,15 @@
+import { getSQLFileData } from "../../utilities";
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CleanRestrictionsTable1652466821171 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Truncate-restriction.sql", "Restrictions"),
```

Comment:
After the migration for clean-up of restrictions, I am not seeing the alter column sql to change the integer to serial for restrictions table.

Reply:
its already serial @dheepak-aot 
![image](https://user-images.githubusercontent.com/77353155/168695216-460284b1-079a-4893-89df-74e13a5b9afe.png)
the fix is don't use id while inserting